documentation/adr/002-trust-github-oidc.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: page.njk
+status: Accepted
+date: git Created
+tags:
+  - adr
+order: 2
+title: 002 - Trust GitHub through OpenID Connect for platform development 
+---
+
+**Date: {{ page.date.toISOString().substring(0,10) }}**
+
+**Status: {{ status }}**
+
+## Context
+
+We have decided in [ADR 1](001-use-aws-landing-zone-accelerator.md) to use LZA.
+
+Each LZA deployment needs source code in a CodeCommit repository in the same account.
+
+We have multiple root accounts each containing an LZA deployment, and we want to be able to manage those environments from a single source code repository.
+
+We need to identify a platform for our developers to maintain our infrastructure and tooling code.
+
+Using CodeCommit is not desirable as we would have to share repositories across different root accounts.
```

Comment:
I think theres something else here about mentioning the effort required to setup users with permission to access all the repositories?

Reply:
Resolved as adding as consequence.

Reply:
I have moved this to consequence